.storybook/main.js

```diff
@@ -4,6 +4,7 @@ const autoprefixer = require('autoprefixer');
 module.exports = {
   stories: [
     '../packages/*/lib/*/*.stories.js',
+    '../packages/*/lib/*.stories.js',
```

Comment:
can't we replace those two lines by

```suggestion
    '../packages/*/lib/**/*.stories.js',
```